### DISCUSSION TITLE
Getting a hydration error when opening the dashboard

### DISCUSSION THREAD

**Discussion by galigutta** [avitorio/outstatic#265] [Q&A] [answered]
just deployed outstatic on my next.js app. It seems to be working fine generally and have been able to create a collection, add a post etc. but when I go to localhost:3000/outstatic, I get the following error:

Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.
See more info here: https://nextjs.org/docs/messages/react-hydration-error

i can dismiss it and keep going without affecting functionality, but it just keeps bugging me each time i get in there. 

I am on the below versions if that helps:
     "next": "^14.2.18",
    "outstatic": "^1.4.11",

**Comment by avitorio** [comment]
Can you add a screenshot of the error here?

**Reply by galigutta** [reply]
![Screenshot from 2024-11-19 19-28-22](https://github.com/user-attachments/assets/b0390411-18a0-475f-ba23-2a74b7a3415f)
![Screenshot from 2024-11-19 19-28-30](https://github.com/user-attachments/assets/4a347e1f-9e4c-4453-8ee7-f8fac2175fb2)

**Reply by galigutta** [reply]
i am not sure if any of the layout.tsx from my main app is interfering with this. I did have a floating component in my landing page that somehow made it to this screen and i suppressed it in my app's layout.tsx. I am a bit of noob with frontend stuff. apologies if user error

**Reply by avitorio** [reply, marked as answer]
I'd advise tou to add your app to a route group, I usually do (web)/ to avoid having your main layout interfere with outstatic... which should live under (cms)/

**Reply by galigutta** [reply]
![image](https://github.com/user-attachments/assets/0c465e2d-eab0-48df-a471-9f8aa94f8c6c)
this is my folder structure fwiw

**Comment by galigutta** [comment]
thank you! that worked like a charm
